src/main/java/org/apache/commons/csv/CSVFormat.java

```diff
@@ -1628,7 +1628,8 @@ private CharSequence trim(final CharSequence charSequence) {
     /**
      * Verifies the consistency of the parameters and throws an IllegalArgumentException if necessary.
      *
-     * @throws IllegalArgumentException
+     * @throws IllegalArgumentException If something goes wrong
```

Comment:
Hi @arturobernalg 
This change does not help anyone, and it's confusing when the line above tells you when the exception would be thrown. I've updated git master for this Javadoc. Please rebase on master. TY.

Reply:
OK. Got it. Changed